service-commands/scripts/whitenoise.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Generates a random whitenoise mp3 file of given input size.
+ *
+ * Script usage: node whitenoise.js <size> <outFile>
+ */
+
+const { spawn } = require('child_process')
+const ffmpeg = require('ffmpeg-static')
+
+async function run() {
+  try {
+    const { size, outFile } = parseArgs()
+    whitenoise(size, outFile)
+  } catch (err) {
+    console.error(err.message)
+  }
+}
+
+/**
+ * Process command line args, defaults to 1 KiB file size, and unix timestamp for filename
+ */
+function parseArgs() {
+  const args = process.argv.slice(2)
```

Comment:
if you want to make this smarter, you can use the npm package Commander to parse your args :) [example](https://github.com/AudiusProject/audius-protocol/pull/1021/files#diff-82561c9e6593b9b7fc9485e24e8c78a13c07225161775c65e9dcd9c2d38d1c27R2)

Reply:
Thanks, that looks much slicker than my parsing! I'll update my code.